test/BUILD.gn

```diff
@@ -855,7 +836,12 @@ if (!is_android) {
       "//ui/compositor:test_support",
       "//ui/views",
     ]
-
+    if (ipfs_enabled) {
+      deps += [
+        "//brave/browser/ipfs/test:browsertests",
+        "//brave/components/ipfs:ipfs"
```

Comment:
this one is not needed.

Reply:
this is needed for some infobar browser tests

Reply:
As @spylogsster pointed out to me, needed by `brave/browser/brave_profile_prefs_browsertest.cc`